src/components/Tabs/BarItem.js

```diff
@@ -7,7 +7,15 @@ import {
 } from '@newrelic/gatsby-theme-newrelic';
 import useTabs from './useTabs';
 
-const BarItem = ({ index, children, id, count, disabled }) => {
+const BarItem = ({
+  index,
+  children,
+  id,
+  count,
+  disabled,
+  quickstartName,
```

Comment:
I feel a little icky about passing all this quickstart specific information into this more general component, even though this is the only page that currently uses it. If we avoid it now, it will be much easier to generalize this component later. That said, this is by no means a required change.
We could keep this component general by passing in a callback for click events, then that callback could be created with all the quickstart specific info in `QuickstartDetails` before even initializing the `BarItem` component.
```js
const tessenTrack = (action, quickstart) => (extraFields) => {
  tessen.track('instantObservability', action, {
    quickstartTitle: quickstart.title,
    quickstartId: quickstart.id,
    ...extraFields,
  };
};
```
Then you could call this with the relevant information and then pass the returned function to the downstream components.

Reply:
Yes totally agree on this one, i've refactored but not sure if that's as tidy as it could be 😬